### DISCUSSION TITLE
How to update credentials on a custom source

### DISCUSSION THREAD

**Discussion by nils-a** [chocolatey/choco#2247] [Q&A] [answered]
Hello,
I have a machine with choco installed and an additional source which needs credentials. Now I need to update the credentials and I was looking for an easy way to do that.

I.e. with nuget I can do `dotnet nuget source update ...` but `chcoc source --help` indicates that there is no update command.

For now I used `choco source remove` and `choco source add` but `update` would be nice. Or am I missing something?

**Comment by TheCakeIsNaOH** [comment, marked as answer]
You can run `choco source add` to change a setting on a source without having to `remove` it first.

Everything does have to be specified again, so `name`, `source`, `priority`, `user`, `pass`, etc.
